src/renderer/modules/components/Slider.tsx

```diff
@@ -0,0 +1,25 @@
+import { filters, waitForModule } from "../webpack";
+
+interface SliderProps {
+  disabled?: boolean;
+  markers?: number[];
+  stickToMarkers?: boolean;
+  initialValue?: number;
+  defaultValue?: number;
+  minValue?: number;
+  maxValue?: number;
+  mini?: boolean;
+  style?: React.CSSProperties;
+  onValueChange?: (e: number) => void;
+  asValueChanges?: (e: number) => void;
+  onValueRender?: (e: number) => string;
+  onMarkerRender?: (e: number) => string;
```

Comment:
Missing the `className` property

Reply:
Fixed in [this commit](https://github.com/replugged-org/replugged/pull/366/commits/42ff71b350430157d78a50f5bce59b144e07d1cb#diff-ba27653ecb680e72e999facdb29940cc49686074aee6280329274baf5d104122R14)